src/gradio_app.py

```diff
@@ -16,480 +16,26 @@
 
 # LIBRAIRIES IMPORT
 import os
-import json
-import random
-from typing import Tuple
 
-import re
 import gradio as gr
 from loguru import logger
 
 
 # MODULES IMPORT
-from create_database import load_documents, get_file_names
-from query_chatbot import (
-    load_database,
-    search_similarity_in_database,
-    get_metadata,
-    format_chat_history,
-    contextualize_question,
-    generate_answer,
-    add_metadata_to_answer,
-    format_relevant_chunks,
-    MSGS_QUERY_NOT_RELATED,
-    OPENAI_MODEL_NAME,
-    CHROMA_PATH,
-)
-from create_quiz import create_quiz_json, get_chapter_content
+from gradio_app_chat import create_tab_discuss_course
```

Comment:
Could you rename `create_tab_discuss_course` as `create_tab_chatbot`?

Reply:
Yes !